docs/technical/rebase-flow.md

```diff
@@ -49,16 +52,25 @@ setup:
  2. find the range of commits from `<upstream_oid>..<branch_oid>`
  3. generate a patch series of the contents, ready to rebase
 
-**TODO:** As part of detecting conflicts in https://github.com/desktop/desktop/issues/6960
-we need some documentation here on how the patches are generated as part of a
-rebase, and how the application emulates and tests this.
+### Detecting Conflicts
+
+**TODO:** As part of detecting conflicts in [#6960](https://github.com/desktop/desktop/issues/6960)
+there will be some words here eventually to help explain how the solution works.
 
 ## Warning about remote commits
 
-**TODO**: write a section about the decisions made here based on what gets
-implemented in https://github.com/desktop/desktop/issues/6963. This needs to
-inspect the tracked branch and see what commits are on the remote, and confirm
-if any of those are in the list of commits that will be rebased.
+If the target branch is tracking a remote branch and the user has enabled the
+_Show confirmation dialog before force pushing_ setting, Desktop
+will check if any commits exist in the range
+`<upstream_oid>..<remote_branch_oid>`, where `<remote_branch_oid>` is the tip of
+the tracking branch.
+
+This check is important to identify any remote commits that will be caught in
+this rebase, because this will require rewriting the history on the remote,
+which the user may not have permission to do based on branch protections.
```

Comment:
```suggestion
which the user may not have permission to do based on branch protections or may not want to do because of the impact it can have on other collaborators.
```